src/util.ts

```diff
@@ -181,3 +183,9 @@ export function encodeHashParameter(params: Record<string, string>): string {
     })
     .join('&');
 }
+
+export function pathStartsWith(path1: string, path2: string): boolean {
+  const path1n = upath.normalizeTrim(path1) + '/';
+  const path2n = upath.normalizeTrim(path2) + '/';
```

Comment:
It seems not to need a trailing slash.

```suggestion
  const path1n = upath.normalizeTrim(path1);
  const path2n = upath.normalizeTrim(path2);
```

Reply:
The trailing slash is necessary, e.g., when `path1` is `"/test/abcdef/test.md"` and `path2` is `"/test/abc"`. Without the trailing slash, `path1n.startsWith(path2n)` returns `true`, that's wrong.

Reply:
Fixed the problem when path2 is the root dir "/": https://github.com/vivliostyle/vivliostyle-cli/pull/149/commits/46c370c1c21035877dc0c7a3463b683070708ce9